src/api/AppActivity.js

```diff
@@ -0,0 +1,144 @@
+// /2.0/app_activities?item_id=<FILE_ID>&item_type=file&fields=occurred_at,rendered_text,app,created_by,activity_template&marker=<NEXT_MARKER>&limit=<MY_LIMIT>
+
+/**
+ * @flow
+ * @file Helper for the box App Activity API
+ * @author Box
+ */
+import MarkerBasedAPI from './MarkerBasedAPI';
+import {
+    ERROR_CODE_DELETE_APP_ACTIVITY,
+    FIELD_ACTIVITY_TEMPLATE,
+    FIELD_APP,
+    FIELD_CREATED_BY,
+    FIELD_OCCURRED_AT,
+    FIELD_RENDERED_TEXT,
+} from '../constants';
+
+const APP_ACTIVITY_FIELDS: string = [
+    FIELD_ACTIVITY_TEMPLATE,
+    FIELD_APP,
+    FIELD_CREATED_BY,
+    FIELD_OCCURRED_AT,
+    FIELD_RENDERED_TEXT,
+].join(',');
+
+/**
+ * Map an entry from the AppActivity API to an AppActivityItem.
+ * occurred_at -> created_at
+ *
+ * @param {Object} item - A single entry in the AppActivity API entiries list
+ */
+function mapAppActivityItem(item: Object): AppActivityItem {
+    const { occurred_at, ...wOutOccurredAt } = item;
```

Comment:
`wOutOccurredAt` is a bit interesting, how about something more traditional like `rest`?